src/gui/aboutdialog.cpp

```diff
@@ -45,7 +45,7 @@ AboutDialog::AboutDialog(QWidget *parent)
     // Title
     m_ui->labelName->setText(QString("<b><h2>qBittorrent " QBT_VERSION " (%1-bit)</h2></b>").arg(QT_POINTER_SIZE * 8));
 
-    m_ui->logo->setPixmap(Utils::Gui::scaledPixmapSvg(":/icons/skin/qbittorrent-tray.svg", this, 32));
+    m_ui->logo->setPixmap(Utils::Gui::scaledPixmapSvg("QBittorrent.Logo", this, 32));
```

Comment:
I would use "AboutDialog.Logo".

Reply:
@glassez:  It is my understanding that the same icon is used in multiple places (including the system tray). 

Do you really want to name the icon based on only one of those components?

Reply:
@LordNyriox, if you had followed #10903, you might have understood my intentions...
https://github.com/qbittorrent/qBittorrent/issues/10903#issuecomment-510820381